package.json

```diff
@@ -25,9 +25,10 @@
                     "cylc"
                 ],
                 "filenamePatterns": [
-                    "suite*.rc",
+                    "*suite*.rc",
                     "suite*.rc.*",
                     "*.cylc",
+                    "*.cylc.*",
```

Comment:
```suggestion
"*.cylc*",
```

Reply:
Are these globs rather than regexes?

Reply:
Was trying to look this up but then realised/remembered that `filenamePatterns` isn't actually documented! But yes, it's globs.

Reply:
Are you aware of any uses of the file pattern `*.cylc*` without a second `.`? If so I'll make the change (and I guess `*suite.rc.* -> *suite.rc*`), but if not then I won't bother 😛